core/src/main/java/org/bitcoinj/wallet/WalletEx.java

```diff
@@ -973,4 +982,37 @@ List<TransactionOutput> getCoinJoinOutputs() {
         }
         return result;
     }
+
+    public String getTransactionReport() {
+        MonetaryFormat format = MonetaryFormat.BTC.noCode();
+        StringBuilder s = new StringBuilder("Transaction History Report");
+        s.append("\n-----------------------------------------------\n");
+
```

Comment:
This function will give us a high level view of transaction history that will display transactions in order by date.